packages/e2e-test-utils/src/click-block-toolbar-button.js

```diff
@@ -8,20 +8,12 @@ import { showBlockToolbar } from './show-block-toolbar';
  *
  * @param {string} buttonAriaLabel The aria label of the button to click.
  */
-export async function clickBlockToolbarButton( buttonAriaLabel ) {
+export async function c( buttonAriaLabel ) {
```

Comment:
Why the name change here? Or is this just a typo?

Reply:
typo :)